test/library/input_numeric_format.spec.js

```diff
@@ -603,6 +603,22 @@ describe('Test NumberFormat as input with numeric format options', () => {
     expect(wrapper.state().value).toEqual('23,4456 Sq. ft');
   });
 
+
+  it('should allow pasting 123,45 with decimal separator set to . and allowedDecimalSeparators=["," , "."]', () => {
```

Comment:
Let's rename this to 
```suggestion
  it('should work with the configured decimal separators based on the allowedDecimalSeparators prop', () => {
```